### ISSUE TITLE
Error when pressing bind key

### ISSUE THREAD

**Issue by anthonysharpy** [civilgamers/bug-reports#563]
**Description** 

Self-explanatory.

**Steps to Reproduce** 

Unclear.

https://rovester.shop/ss/auqlctcpfi3rgnxy.png

**Expected Behavior** 

**Evidence** 

https://rovester.shop/ss/anl55l1iexwnrhm6.png

**Additional Context** 

Reported by Rovester. Maybe it's caused by the fact that the Y key is also the chat key? In that case we should disable the Y key for binds.

**Comment by anthonysharpy** [comment]
#549 ? Possibly linked?

**Comment by anthonysharpy** [comment]
Rovester says it only happens for the Y key.

**Comment by Cerbiac** [comment]
Yeah, I already said that it refers to VBinds, as they'll conflict with normal binds. The source engine won't let you use binds while typing.

**Comment by anthonysharpy** [comment]
I meant that Rovester said it only happens when the bind is assigned to the Y key. Not sure if I was clear/if I'm understanding correctly.
